docs/calamari/governance_overview.md

```diff
@@ -0,0 +1,91 @@
+
+## Launch Period
+Changes to the Calamari system start out as a **proposal**. Token holders 
+or the Council submit the [preimage hash of a proposal](https://wiki.polkadot.network/docs/maintain-guides-democracy#proposing-an-action) 
+to the proposal queue. Other token holders can then second the proposal with 
+their stake, and at the end of a Launch Period the proposal with the most 
+backing stake is promoted to a **referendum**. We only promote one proposal 
+at a time to avoid the situation where two opposing proposals pass simultaneously.
+
+Right now, only the council can submit new proposals. The method they use 
+is called `democracy.externalProposeDefault`, and it will be available to
+all token holders in upcoming releases. 
+
+
+## Voting Period
+All token holders get to vote aye or nay on referendums. The percent of ayes required
+for a referrendum to pass changes based on turnout. The method `democracy.externalProposeDefault`
+has a "negative turnout bias", meaning if there's low turnout, the 
+threshold to pass is lower, and as turnout increases, the threshold increases. 
+For example, if the turnout is 25%, then 34% ayes are needed to pass. But if the 
+turnout is 100%, then 51% of ayes are needed (a simple majority). 
+
+This approach to voting is called [Adaptive Quorum Biasing](https://wiki.polkadot.network/docs/learn-governance#adaptive-quorum-biasing). 
+The idea behind it is to anticipate that voter turnout in practice is
+never going to be 100%, and that different proposals will have varying levels of 
+contientiousness and trustworthiness. For example, we assume that a proposal 
+submittes by the Council is more trust worthy than a proposal from any public 
+token holder, so we want to apply a negative turnout bias to the vote.
+
+## Enactment Period
+Once a referendum passes, it is automatically implemented by the chain according to 
+the referendum code. There's a delay before the implementation starts so that
+stake holders can prepare for the change. 
+
+### A note on the responsibilities of the Council and Technical Committee
+The Technical Committee can fast-track the launch period, voting period, and enactment
+period in the case of an emergency. How much the periods are shortened depends 
+on how many committee members believe the action is necessary, with unanimous 
+agreement required for instantaneous changes. 
+
+The Council can cancel a referendum during the voting period. TODO: any other 
+of these sorts of actions that the council can do?
+
+Clearly the Council and the Technical Committe have a lot of influence over
+what proposals and referrendums get passed! The Council offsets the usual low 
+voter turnout in democratic systems, and is expected to submit proposals for 
+regular operations and maintenance of the system. The Technical Committe exists
+to handle emergency situations where fast technical updates are needed.
+
+## How To: Vote on a Referendum
+TODO: include screenshots
```

Comment:
@stechu I think we should include screenshots on how to vote on a referendum but I'm not sure where that lives, if you point me in the right direction I can try to do that, or if we already have some we can link that here.

Reply:
will do. we can do a dry run and get some screen shots.